### ISSUE TITLE
Time component a11y violation

### ISSUE THREAD

**Issue by InnaAtanasova** [SAP/fundamental-styles#1152]
Storybook shows an a11y violation for Time component:
<img width="718" alt="Screen Shot 2020-06-12 at 3 59 56 PM" src="https://user-images.githubusercontent.com/39598672/84541831-260b5380-acc6-11ea-9666-91936f0a470a.png">

The fix requires introducing Breaking Changes since fixing it would require changing the markup
